audience_manager/authorized_api/paths/audiences_list.yaml

```diff
@@ -25,19 +25,22 @@ get:
   - name: name
     type: string
     in: query
-    description: Filter results by a fragment of its name (case insensitive).
+    description: Searched name - matches names containing this string (case insensitive).
   - name: order_by
     type: string
     in: query
     description: >
-      Sorting order for the results. Include <code>-</code> prefix in value
-      to get a descending order.
+      Sorting order for the results. Values with `-` prefix return results in descending order.
     default: name
     enum:
     - name
+    - -name
     - count
+    - -count
```

Comment:
How it works with two `- -`?

Reply:
It is treating it as part of the string, so we have `"count", "-count", ...`